packages/form-wrapper/src/FormWrapperFooter.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Button, { buttonTypes, buttonSizes } from "@crave/farmblocks-button";
+import styled from "styled-components";
+import { colors } from "@crave/farmblocks-theme";
+
+export const Footer = styled.div`
+  padding: 16px;
+  border-top: 1px solid ${colors.GREY_16};
+`;
+
+const FormWrapperFooter = ({
+  id,
+  loading,
+  onSave,
+  saveButtonProps,
+  saveLabel,
+}) => (
+  <Footer className="footer">
+    <Button
+      className="save"
+      id={`${id}-save`}
+      type={buttonTypes.SECONDARY}
+      size={buttonSizes.MEDIUM}
+      onClick={onSave}
+      loading={loading}
+      fluid
+      {...saveButtonProps}
+    >
+      {saveLabel}
+    </Button>
+  </Footer>
+);
+
+FormWrapperFooter.defaulProps = {
+  cancelLabel: "Cancel",
```

Comment:
either add a default `saveLabel` here or make it required

Reply:
Actually this cancelLabel shouldn't be here, it should be on FormWrapperHeader instead and vice-versa